### PR TITLE
docs: WIP: openapi monitoring guide

### PR DESCRIPTION
**Pull request by ragog**

### CODE REVIEW

File: site/content/guides/openapi-swagger.md [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+title: OpenAPI/Swagger Monitoring
+description: >-
+  OpenAPI and Swagger help users design and document APIs in a way that is readable from both humans and machines. As a consequence, they can also be used to generate the code that will run the specified API - both on the provider and consumer side. Can we leverage this same principle to simplify API monitoring?
+author: Giovanni Rago
+avatar: 'images/avatars/giovanni-rago.png'
+---
+
+## The OpenAPI Specification
+
+The [OpenAPI Specification (OAS)](https://spec.openapis.org/oas/v3.1.0) specifies a standard, language-agnostic and machine-readable format to describe a web API in one or more files. Nowadays, it acts as a vendor-neutral standard for describing the structure and behaviour of HTTP-based APIs, and exists as part of the [OpenAPI Initiative (OAI)](https://www.openapis.org/).
```
Comment (hlenke) [inline site/content/guides/openapi-swagger.md:11]: Do we need an intro? Something like.
First, we are looking at OpenApi and afterward at how to kickstart your monitoring with existing OpenApi specs.

File: site/content/guides/openapi-swagger.md [hunk 2]
```diff
@@ -0,0 +1,230 @@
...
+
+We can open the newly created checks and change any of their settings.
+
+{{< figure src="/guides/images/guides-checkly-openapi-body.png" alt="screenshot of checkly filled in check body" >}}
+
+After a small amount of tinkering, or none at all, we should have all our checks up and running.
+
+{{< figure src="/guides/images/guides-checkly-openapi-pass.png" alt="screenshot of checkly checks - all passing" >}}
+
+For APIs medium and large, and for any that is already described with an OAS file, this check-generating procedure can save us a large amount of time.
```
Comment (hlenke) [inline site/content/guides/openapi-swagger.md:222]: Good article.
General comments:
1. external links need **_target=_blank_**
2. Can you be a bit more vocal about the benefits of creating checks via OpenAPI spec? Also should I create checks for every endpoint that is defined in the OpenAPI spec?

**Review by hlenke — CHANGES_REQUESTED** [review]

File: site/content/guides/openapi-swagger.md [hunk 3]
```diff
@@ -0,0 +1,235 @@
...
+
+  ...
+
+```
+
+Depending on how much is described in a given API spec, it is possible to move even further and generate tests, mock servers and more. See <a href="https://openapi.tools" target=_blank>openapi.tools</a> for an up-to-date list of tools belonging to the OpenAPI ecosystem.
+
+## Generated monitoring checks with Checkly
+
+API monitoring is key to ensure that our endpoints are returning the correct results in an acceptable timeframe.  If we can generate APIs from a OAS description file, we can also generate monitoring checks for them. Checkly is [part of the OpenAPI initiative](), and allows us to import an OpenAPI spec file and automatically creates one or more checks depending on the amount of information available. All it takes is a couple of clicks.
```
Comment (ragog) [inline site/content/guides/openapi-swagger.md:190]: @Haubersch here I would link the announcement from OpenAPI. We can also have this pop out a bit more if needed.

File: site/content/guides/openapi-swagger.md (hunk 1 above)
Comment (ragog) [reply site/content/guides/openapi-swagger.md]: Thanks, added this.

File: site/content/guides/openapi-swagger.md (hunk 2 above)
Comment (ragog) [reply site/content/guides/openapi-swagger.md]: 1. Done. @tnolet maybe you know of a way to make this work with markdown? My little research says I either have to go with pure HTML (which I have done this time) or install some other markdown-like package.
2. Done.

Comment (hlenke) [reply site/content/guides/openapi-swagger.md]: @ragog on 1) https://adityatelange.in/blog/hugo-external-link-new-tab/

Comment (ragog) [reply site/content/guides/openapi-swagger.md]: @hlenke good enough. I wish there was a pure markdown solution for this :) thanks
I am opening a new PR and converting all previous guides links.

File: site/content/guides/openapi-swagger.md (hunk 3 above)
Comment (ragog) [reply site/content/guides/openapi-swagger.md]: Applying.

**Review by hlenke — APPROVED** [review]

**Pull request merged**